src/ape_geth/provider.py

```diff
@@ -197,9 +199,9 @@ def wait(self, *args, **kwargs):
 
 class GethNetworkConfig(PluginConfig):
     # Make sure you are running the right networks when you try for these
-    mainnet: Dict = DEFAULT_SETTINGS.copy()
-    goerli: Dict = DEFAULT_SETTINGS.copy()
-    sepolia: Dict = DEFAULT_SETTINGS.copy()
+    mainnet: Dict = {"uri": random.choice(PUBLIC_CHAIN_RPCS["ethereum"]["mainnet"])}
+    goerli: Dict = {"uri": random.choice(PUBLIC_CHAIN_RPCS["ethereum"]["goerli"])}
+    sepolia: Dict = {"uri": random.choice(PUBLIC_CHAIN_RPCS["ethereum"]["sepolia"])}
     # Make sure to run via `geth --dev` (or similar)
     local: Dict = {**DEFAULT_SETTINGS.copy(), "chain_id": DEFAULT_TEST_CHAIN_ID}
```

Comment:
i think here was have to add a custom validator again or something to check for any network that isn't part of Ethereum